iaso/api/mobile/metadata/last_updates.py

```diff
@@ -0,0 +1,75 @@
+from django.db.models import Max
+from django.shortcuts import get_object_or_404
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status, permissions
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from iaso.api.query_params import APP_ID
+from iaso.models import Form, FormVersion, OrgUnit, OrgUnitType, Project
+
+
+class LastUpdatesViewSet(ViewSet):
+    """Metadata Last Updates
+
+    This API is open to anonymous users:
+
+    /api/mobile/metadata/lastupdates/
+
+    `GET /api/mobile/metadata/lastupdates/?app_id=some.app.id`
+    """
+
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
+    http_method_names = ["get", "head", "options"]
+    lookup_url_kwarg = [APP_ID]
+
+    app_id_param = openapi.Parameter(
+        name=APP_ID,
+        in_=openapi.IN_QUERY,
+        required=True,
```

Comment:
@bmonjoie I thought it was not required?

Reply:
Oups :)